engineio/context.go

```diff
@@ -0,0 +1,29 @@
+package engineio
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/googollee/go-socket.io/engineio/packet"
+)
+
+// Context provides info for a HTTP request.
+type Context struct {
+	// Standard context
+	context.Context
+
+	// The instance of current session.
+	Session Session
+
+	// The http.Request of current HTTP request.
+	// It may be different when using polling transport.
+	Request *http.Request
+
+	// The current packet infomation.
+	PacketType packet.Type
+	Reader     io.Reader
+}
+
+// Next calls following middlewares in engine.io framework.
+func (c *Context) Next() {}
```

Comment:
i think it is raw signature and will be changed in new version

Reply:
Which name do you want to change to?

Reply:
sorry, it is good signature